src/components/Tokens/TokenDetails/About.tsx

```diff
@@ -24,8 +25,8 @@ const TokenDescriptionContainer = styled.div`
 const TruncateDescriptionButton = styled.div`
   color: ${({ theme }) => theme.textSecondary};
   font-weight: 400;
-  font-size: 14px;
-  padding-top: 14px;
+  font-size: 0.85em;
```

Comment:
i do want to settle what we should do here with units but cool with this for now

Reply:
haha, thought of you as I wrote this